web_src/js/features/notification.js

```diff
@@ -120,8 +117,12 @@ export function initNotificationCount() {
   startPeriodicPoller(notificationSettings.MinTimeout);
 }
 
+function getCurrentCount() {
+  return document.querySelector('.notification_count').textContent;
+}
+
 async function updateNotificationCountWithCallback(callback, timeout, lastCount) {
-  const currentCount = $('.notification_count').text();
+  const currentCount = getCurrentCount();
```

Comment:
There was a actual bug here because there are two `.notification_count` elements on the page, in which case jQuery concats the result, e.g. `"00"` for 0 notifications and `"11"` for one.

Reply:
Yes, it is fallback code. It only falls back when the browser does not support SSE, which is likely a given since many years now. We should rewrite this whole SSE thing to use websockets, that depends on https://github.com/go-gitea/gitea/pull/20543 mostly.